L/libgraphicsmagic/build_tarballs.jl

```diff
@@ -34,8 +34,19 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="libwebp_jll", uuid="c5f90fcd-3b7e-5836-afba-fc50a0988cb2"))
+    Dependency(PackageSpec(name="libpng_jll", uuid="b53b4c65-9356-5827-b1ea-8c7a1a84506f"))
```

Comment:
Hold on, today I discovered an incompatibility in libpng, you'll have to specify the version

Reply:
does this work now?

Reply:
No, it'll work after I'll merge https://github.com/JuliaRegistries/General/pull/37269 (which will happen after I'll fix all the compat bounds everywhere)

Reply:
ok, no rush, just merge whenever the rest of the landscape is ready. This is only used by my `timg` anyways, and I'm not in a rush.

Reply:
For the record, with #3054 I managed to not break the ABI of the libraries, so specifying the latest version isn't necessary, 1.37 is fine (if you want to specify any version at all, not needed)